rootfs_linux.go

```diff
@@ -100,7 +103,26 @@ func mount(m *configs.Mount, rootfs, mountLabel string) error {
 			return err
 		}
 		return syscall.Mount(m.Source, dest, m.Device, uintptr(m.Flags), "")
-	case "tmpfs", "mqueue", "devpts", "sysfs":
+	case "tmpfs":
+		if stat, err = os.Stat(dest); err == nil {
+			if err := os.MkdirAll(dest, 0755); err != nil && !os.IsExist(err) {
+				return err
+			}
+		}
+		if err := syscall.Mount(m.Source, dest, m.Device, uintptr(m.Flags), data); err != nil {
+			return err
+		}
+		if stat != nil {
+			if file, err = os.Open(dest); err != nil {
+				return err
+			}
+			defer file.Close()
+			if err = file.Chmod(stat.Mode()); err != nil {
+				return err
+			}
```

Comment:
can we use `os.Chmod(dest, stat.Mode())` here instead of having to open the file and use fchmod ?